code/modules/scrap/scrap.dm

```diff
@@ -252,6 +253,9 @@ GLOBAL_LIST_EMPTY(scrap_base_cache)
 	user.setClickCooldown(DEFAULT_ATTACK_COOLDOWN)
 	if(hurt_hand(user))
 		return
+	prob_old_item = initial(prob_old_item)
+	if(user.stats.getPerk(PERK_JUNKBORN))
+		prob_old_item = 80
```

Comment:
This is wrong, this logic makes items worse actually. What it supposed to do - is to spawn rare item using a random spawner before deleting a trashpile, if it was removed by a person with a Junkborn perk.